src/context/TransactionsProvider/TransactionsProvider.tsx

```diff
@@ -58,7 +61,7 @@ export const TransactionsProvider = ({ children }: TransactionsProviderProps): J
     // account specifiers changing will trigger this effect
     // we've disconnected/switched a wallet, unsubscribe from tx history and clear tx history
     if (!isSubscribed) return
-    console.info('TransactionsProvider: unsubscribing from tx history')
+    moduleLogger.info('TransactionsProvider: unsubscribing from tx history')
```

Comment:
You can remove Transactionprovider from the message since it's in the namespace now

Reply:
derrrp